src/panel.ts

```diff
@@ -1830,6 +1830,11 @@ export class PanelModel extends PanelModelBase
       }
     });
   }
+
+  public isRenderOldFooter = true;
```

Comment:
How to set new footer rendering for preview?

Reply:
lets discuss it with @andrewtelnov  and @tsv2013